src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -582,17 +582,12 @@ export function fetchConfirmedAppointmentDetails(id, type) {
         });
       }
 
+      // We would expect to have the clinic name here, but if we don't, fetch it
       if (
-        featureVAOSServiceCCAppointments &&
```

Comment:
Just removing this fetch altogether, since we won't need it for the confirmed CC details.